dynawo/sources/Models/CPP/ModelNetwork/DYNModelDanglingLine.cpp

```diff
@@ -89,17 +89,22 @@ modelType_("DanglingLine")  {
 
 
   double factorPuToA = sqrt(3.) * vNom / (1000. * SNREF);
+  double limit = 0.;
```

Comment:
let's declare it only when used (inside the if)

Reply:
Corrected